server/__test__/index.test.js

```diff
@@ -350,6 +350,15 @@ describe('authentication', () => {
       });
     });
 
+    describe('Test Get /logout', () => {
+      test('should return successfully logout with status 200 with message logged out successfully', async () => {
+        const res = await request(app)
+          .get('/api/v1/logout')
+          .set('Cookie', [`token=${token}`]);
+        expect(res.status).toBe(200);
+        return expect(res.body.message).toBe('logged out successfully');
```

Comment:
Is it possible to add an expert statement to verify that the token cookie has been cleared? Then this test would guarantee that the logout endpoint was actually logging users out